File: Tech Module/ST - Exams/ShoppingList/C# Solution/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace ShoppingList.Models
{
    public class Product
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int Priority { get; set; }

        [Required]
        [MinLength(1)]
        public string Name { get; set; }

        [Required]
        public int Quantity { get; set; }

        [Required]
        public string Status { get; set; }
    }
}

File: Tech Module/ST - Exams/ShoppingList/JavaScript Solution/models/Product.js
const mongoose = require('mongoose');

let productSchema = mongoose.Schema({
    priority: {type: 'number', required: 'true'},
    name: {type: 'string', required: 'true', minlength: 1},
    quantity: {type: 'number', required: 'true'},
    status: {type: 'string', required: 'true', minlength: 1},
});

let Product = mongoose.model('Product', productSchema);

module.exports = Product;